packages/docusaurus-theme-classic/src/theme/SiteMetadata/index.tsx

```diff
@@ -33,16 +33,22 @@ function AlternateLangHeaders(): JSX.Element {
   return (
     <Head>
       {Object.entries(localeConfigs).map(([locale, {htmlLang}]) => (
-        <link
-          key={locale}
-          rel="alternate"
-          href={alternatePageUtils.createUrl({
-            locale,
-            fullyQualified: true,
-          })}
-          hrefLang={htmlLang}
-        />
+		    <>
+		      <link
+            key={locale}
+            rel="alternate"
+            href={alternatePageUtils.createUrl({
+              locale,
+              fullyQualified: true,
+            })}
+            hrefLang={htmlLang}
+          />
+		      <meta property="og:locale:alternate" content={locale} />
```

Comment:
Same, we should use the locale bcp47 code (`htmlLang` attribute like in hreflang links), but converted to og locale format (see https://github.com/facebook/docusaurus/issues/8887#issuecomment-1514981713)

Also, it seems OpenGraph does not recommend outputting the current locale as an alternate

```html
<meta property="og:locale" content="en_GB" /> 
<meta property="og:locale:alternate" content="fr_FR" />
<meta property="og:locale:alternate" content="es_ES" />
<meta property="og:locale:alternate" content="en_GB" /> ❌
```